src/sentry/receivers/sentry_apps.py

```diff
@@ -75,6 +78,30 @@ def send_comment_webhooks(organization, issue, user, event, data=None):
             user_id=(user.id if user else None),
             data=data,
         )
+    if features.has("organizations:sentry-functions", organization, actor=user):
+        from sentry.models import SentryFunction
+
+        data["user"] = serialize(User.objects.get(id=user.id), user, UserSerializer())
```

Comment:
@vpandiarajan20 we should eventually add some logic in case the objects can't be fetched but we can handle that later.